glooshot/docs/tutorial/bookinfo_tutorial.md

```diff
@@ -0,0 +1,265 @@
+---
+title: Bookinfo
+menuTitle: Bookinfo
+weight: 1
+---
+
+## Bookinfo Tutorial
+
+This tutorial will show you how to use Glooshot to apply chaos experiments to a simple service mesh app.
+We will use a slight modification of the familiar bookinfo app from Istio's
+[sample app repo](https://github.com/istio/istio/tree/master/samples/bookinfo). We have modified the reviews service to
+include a vulnerability that can lead to cascading failure. We will use Glooshot to detect this weakness.
+
+
+#### The Goal
+
+Services should be resilient to system outages. In this example, we show how to detect cascading failures: failures
+where an error in one service disables other services that interact with it. In the diagram below, we show two versions
+of a reviews service. The version on the top right fails when it does not receive a valid response from the ratings.
+The version on the bottom right handles the error more gracefully. It still provides review information even though the
+ratings data is not available.
+
+![bookinfo resilience overview](../bookinfo_resilience_demo.png "bookinfo reslience demo")
+
+### Prerequisites
+
+To follow this demo, you will need the following:
+- glooshot v0.0.2 or greater [(download)](https://github.com/solo-io/glooshot/releases)
+- [Supergloo](supergloo.solo.io) v0.3.18 or greater [(download)](https://github.com/solo-io/supergloo/releases)
+- A Kubernetes cluster - [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/#install-minikube) will do
```

Comment:
This list doesn't render as expected

Also, I think we should make clear which of these components can be installed independently of others. For example, you specifically call out Gloo Shot and SuperGloo, but for me, SuperGloo was installed with Gloo Shot.

Reply:
good point, I updated to specify the need for the command line tool